docs/development/pigs/pig-024.rst

```diff
@@ -0,0 +1,283 @@
+authorisation of users to be cited with their personal information (typically, their affiliation and their ORCID if any).
+For this reason, we propose to use the GitHub `Developer Certificate of Origin <https://github.com/apps/dco>`_ (DCO) on
+Pull Request. Its acceptation by contributors will permit to:
+
+- certify that a user wrote or otherwise has the right to submit a code under our open source licence,
+- allow our project to use their personal data for the contributions' record (ie for releases).
+
+The file ``README.md`` will contain a `Contributing` section explaining that the project follows the GitHub DCO, as well
+as a link to this PIG.
+
+.. important::
+    We strongly recommend that the contributors use always the same GitHub account with a valid full name, to
+    synchronise their git email addresses with their GitHub addresses, to have a unique `ORCID <https://orcid.org/>`_
+    identifier and store it into their GitHub user profile for commodity, and inform the developers of any change of
+    affiliation.
+
+
+Definition of the `Maintainer`
+------------------------------
+
+Usually, a software maintainer or package maintainer is one or more people who build source code into a binary package
+for distribution, commit patches, or organize code in a source repository. In the Open Source community, they are
+generally mentioned in the metadata associated with a release.
+
+For Gammapy, by default, the maintainers are the Lead Developers. If in the future a task is dedicated to the creation
+of a release, the maintainer will be the person in charge of this task.
+
+Releases
+--------
+
+Each Gammapy release should be associated with an updated list of authors that will be public on repository (Zenodo and
+HAL/SW). This section is about feature releases. Bug releases will have as list of authors the one of its associated
+feature release whose rules are described here, plus the potentiel additional contributors.
+
+The list of authors is composed of people who contributed to the current release (see :ref:`_contributor`) and who have
+accepted the GitHub DCO. If the DCO is not accepted by a contributor, her/his contribution might not be usable and thus
+this user will not be in the list of authors per default.
+
+The order of the authors is first 'The Gammapy team', followed by the list of contributors in alphabetical order:
+
+ ::
+
+  The Gammapy team: aa, bb, cc, ...
+
+As mentioned in the `PIG 23 <https://github.com/gammapy/gammapy/blob/master/docs/development/pigs/pig-023.rst>`_, the
+publication of a release is preceded by a code freeze period. With its announcement, a call will be made to contributors
+to update their personal information (e.g. their affiliation) under their sole responsibility. This call will also permit
+to add any additional author into the list for her/his special contribution. The Project Managers and Lead Developers
```

Comment:
The his/her is in most modern English text now replaced by just using `they` or `them`, which eases the reading I think and proponents argue does not exclude non-binary people. 

https://en.wikipedia.org/wiki/Singular_they

Reply:
Actually, the sentence above already makes use of it (e.g. their sole responsibility), so let's just be consistent.